client/styles/styles.css

```diff
@@ -270,6 +270,92 @@ body {
   color: #ffffff;
 }
 
+/* CAROUSEL-COMPONENT */
+
+.homepage__carousel {
+  background-color: #ffffff;
+  text-align: center;
+}
+
+.carousel-inner {
+  border-radius: 20px 20px 0 0;
+}
+
+.carousel__title {
+  font-family: Poppins;
+  text-align: center;
+  font-size: 36px;
+  font-weight: bolder;
+  color: #1C2294;
+  margin-top:20px;
+}
+
+.carousel__subtittle {
+  text-align: center;
+  font-family: Poppins;
+  font-size: 48px;
+  font-weight: bolder;
+  color: #1C2294;
+  margin-bottom: 30px;;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100rem;
```

Comment:
For a better responsive design (in combination with padding added above)
```
  width: 100%;
  max-width: 1024px;
  margin-left:auto;
  margin-right: auto;

```

Reply:
I trust you, so I hope it works because yesterday this subject was driving me crazy

Reply:
Does not work (again)